c/bitvector/s3_srvr_2a_alt.BV.c.cil.c

```diff
@@ -0,0 +1,687 @@
+extern void __VERIFIER_error() __attribute__ ((__noreturn__));
+
+extern long __VERIFIER_nondet_long(void);
+extern int __VERIFIER_nondet_int(void);
+/* Generated by CIL v. 1.3.7 */
+/* print_CIL_Input is true */
+
+int ssl3_accept(int initial_state ) 
+{ int s__info_callback  = __VERIFIER_nondet_int();
+  unsigned int s__in_handshake  = __VERIFIER_nondet_int();
```

Comment:
I would use `__VERIFIER_nondet_uint` here, though it should be casted correctly. Did you check the tasks, e.g. with CPAchecker, in order to see whether there are no other overflows?

Reply:
I have not changed  `__VERIFIER_nondet_int` to `__VERIFIER_nondet_uint` in the fixed tasks in general. Should I adapt this?
Yes, I have checked all tasks with CPAchecker locally and in the verifier cloud.

Reply:
Maybe just for the line/variable where you changed the type